@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf

```diff
@@ -122,29 +127,43 @@ if (_velXY < 12.35) then {  //must be less than ETL
         setCamShakeParams [0.0, 0.5, 0.0, 0.0, true];
         addCamShake       [2.5, 1, 5];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
+        SetCustomSoundController[_heli, "CustomSoundController4", 1.8];
```

Comment:
I could see us using custom sound controllers a lot in the future.

Let's take a look and see if we can put each number / name in a preprocessor macro and share it, similar to MFD indexes, to make it easier to read

Reply:
Sounds good to me